app/javascript/mastodon/components/icon_button.js

```diff
@@ -123,6 +125,20 @@ export default class IconButton extends React.PureComponent {
       style.width = 'auto';
     }
 
+    let contents = (
+      <React.Fragment>
+        <Icon id={icon} fixedWidth aria-hidden='true' /> {typeof counter !== 'undefined' && <span className='icon-button__counter'><AnimatedNumber value={counter} obfuscate={obfuscateCount} /></span>}
+      </React.Fragment>
+    );
+
+    if (href) {
+      contents = (
+        <a href={href} target='_blank' rel='noopener noreferrer'>
+          {contents}
+        </a>
+      );
+    }
```

Comment:
Having a link *inside* a button is weird, but it's the only way I have found to have the same padding than on buttons, as for some reason, Firefox seems to put some extra padding inside buttons, that I could not influence no matter what attribute I tried overriding.